client/go.mod

```diff
@@ -1,58 +1,53 @@
 module github.com/kubernetes-csi/external-snapshotter/client/v6
 
-go 1.17
+go 1.18
 
 require (
 	k8s.io/api v0.24.3
 	k8s.io/apimachinery v0.24.3
 	k8s.io/client-go v0.24.3
-	k8s.io/code-generator v0.24.3
```

Comment:
Since you are here, can you also update all the other K8s dependencies?

	k8s.io/api v0.24.3
	k8s.io/apimachinery v0.24.3
	k8s.io/client-go v0.24.3

Reply:
Done!